apps/src/code-studio/pd/application_dashboard/detail_view_contents.jsx

```diff
@@ -4,7 +4,7 @@ import Facilitator1819Questions from './detail_view_facilitator_specific_compone
 import $ from 'jquery';
 import DetailViewResponse from './detail_view_response';
 
-const STATUSES = ['Accepted', 'Waitlisted', 'Pending', 'Declined', 'Unreviewed'];
```

Comment:
There was a request to add "Waiting for interview" here I thought

Reply:
Yes! I'm doing this is a separate PR and also adding 'withdrawn'

Reply:
@mehalshah here is that other PR: https://github.com/code-dot-org/code-dot-org/pull/18855